mappings/net/minecraft/command/argument/BlockArgumentParser.mapping

```diff
@@ -23,16 +26,37 @@ CLASS net/minecraft/unmapped/C_byxnvdet net/minecraft/command/argument/BlockArgu
 	FIELD f_xmojkqxl TAG_PREFIX C
 	FIELD f_yxmqvjdm PROPERTY_DEFINER C
 	FIELD f_yzbucctp blockProperties Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/unmapped/C_vtbxyypo;Lcom/mojang/brigadier/StringReader;ZZ)V
+		ARG 1 blocks
+		ARG 2 reader
+		ARG 3 allowTag
+		ARG 4 allowNbt
 	METHOD m_arwtwfiw parse ()V
+	METHOD m_aswhxtcr parseForBlock (Lnet/minecraft/unmapped/C_tqxyjqsk;Ljava/lang/String;Z)Lnet/minecraft/unmapped/C_byxnvdet$C_lhnnvezv;
+		ARG 0 blocks
+		ARG 1 id
+		ARG 2 allowNbt
 	METHOD m_bhfnnfhg suggestEqualsCharacter (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 builder
 	METHOD m_bniujfgd parseTagId ()V
+	METHOD m_cjscirll parsingForTesting (Lnet/minecraft/unmapped/C_tqxyjqsk;Lcom/mojang/brigadier/StringReader;Z)Lcom/mojang/datafixers/util/Either;
```

Comment:
Why is this `parsing` while the method below is just `parse`?

Reply:
Because I have immeasurable skill

Reply:
Fixed